examples/server/CMakeLists.txt

```diff
@@ -34,14 +34,6 @@ endforeach()
 add_executable(${TARGET} ${TARGET_SRCS})
 install(TARGETS ${TARGET} RUNTIME)
 
-# clean up generated files in pre-build step
```

Comment:
just a note here, we should add a check in `/scripts/xxd.cmake` to see if the file need to be re-generated or not. I will do that in another PR.

Reply:
Ok. You mentioned that the `/slot` endpoint is also broken. I haven't looked at it yet. Maybe we can apply any additional fixes in this PR before merging? Feel free to push directly.

Reply:
Yup I fixed it in https://github.com/ggerganov/llama.cpp/pull/10704/commits/01da1ed9b6d56e656acb98bac7d3d85d2819a2d5

I also fix a problem with cpp wrapper `llama_get_chat_template` because it returns null terminator in the final json:
![Screenshot 2024-12-07 at 16 31 46](https://github.com/user-attachments/assets/6204fbd5-8350-47a5-a980-9e9b33b473e7)